service/application/src/test/java/com/solace/maas/ep/event/management/agent/testConfigs/MessagingServiceTestConfig.java

```diff
@@ -49,6 +55,31 @@ public MessagingServiceEntityToEventConverter entityToEventConverter() {
         return new MessagingServiceEntityToEventConverter();
     }
 
+    @Bean
+    @Primary
+    public MessagingService messagingService() {
```

Comment:
to aid in various tests related to `PersistentMessageReceiver` and `directMessageReceiver`